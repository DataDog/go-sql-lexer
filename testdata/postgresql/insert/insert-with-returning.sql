INSERT INTO users (name, email) VALUES ('Alice Jones', 'alice@example.com') RETURNING id;
