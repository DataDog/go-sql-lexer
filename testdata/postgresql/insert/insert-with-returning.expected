[
    {
        "expected": "INSERT INTO users ( name, email ) VALUES ( ? ) RETURNING id",
        "statement_metadata": {
            "size": 11,
            "tables": [
                "users"
            ],
            "commands": [
                "INSERT"
            ],
            "comments": [],
            "procedures": []
        }
    }
]